### PR TITLE
Controllers specs

### PR DESCRIPTION
**Pull request by initseis**
## Description
In this branch I:

- Added RSpec as a gem to your project.
- Created a Request spec file for all your controllers.
- Made sure that for each action I checked:
[ ] If response status was correct.
[ ] If a correct template was rendered.
[ ] If the response body includes correct placeholder text.

## How Has This Been Tested?
- This has been tested with RSpec Rails

## Screenshots

image

### CODE REVIEW

File: spec/controllers/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
```
Comment (arslanbisharat) [inline spec/controllers/posts_spec.rb]: -  [ ] post_spect.rb should be present in `spec `directroy.

File: spec/controllers/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
```
Comment (arslanbisharat) [inline spec/controllers/users_spec.rb]: -  [ ] users_spect.rb should be present in `spec `directroy.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉 

## Hi @sevinchek, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by initseis** [comment]
Thanks, @DeliceLydia @arslanbisharat.

**Pull request merged**